install_files/ansible-base/roles/ossec/tasks/configure_server.yml

```diff
@@ -21,11 +25,13 @@
   command: >
     gpg
     --homedir /var/ossec/.gnupg
-    --import /var/ossec/{{ ossec_alert_gpg_public_key }}
+    --import /var/ossec/{{ item }}
   become: yes
   become_user: "{{ ossec_group }}"
-  register: add_ossec_gpg_key_result
-  changed_when: "'imported: 1' in add_ossec_gpg_key_result.stderr"
+  when: "'{{ item }}' != ''"
```

Comment:
If I understand, this is opt-in. If `site-specific` does not contain an entry for `journalist_alert_gpg_public_key`, it should throw an error such as `journalist_alert_gpg_public_key' is undefined`. Perhaps this statement should be replaced with something like ` {{ item }} is defined and item != '' `

Reply:
Good catch ! I'll fix this.

Reply:
Two quick notes: 

1. `when` can accept a list, such that _all_ listed statements must evaluate to true for the task to execute
2. `when` statements should _not_ include Jinja curly braces—there's an Ansible deprecation warning to this effect now.